build.sbt

```diff
@@ -33,15 +33,16 @@ libraryDependencies ++= Seq(
   "com.typesafe.akka" %% "akka-actor" % "2.3.4",
   "com.typesafe.akka" %% "akka-slf4j" % "2.3.11",
   "com.typesafe" % "config" % "1.2.1",
-  "com.typesafe.scala-logging" %% "scala-logging" % "3.5.0",
+  "com.typesafe.scala-logging" %% "scala-logging" % "3.7.1",
   "com.typesafe.slick" %% "slick" % "3.2.0",
   "com.zaxxer" % "HikariCP" % "2.3.9",
   "io.spray" %% "spray-can" % sprayV,
   "io.spray" %% "spray-client" % sprayV,
   "io.spray" %% "spray-json" % "1.3.3", // NB: Not at sprayV. 1.3.4 does not exist.
   "io.spray" %% "spray-routing-shapeless23" % sprayV,
   "mysql" % "mysql-connector-java" % "5.1.42",
-  "org.broadinstitute" %% "wdl4s" % wdl4sV,
+  "org.broadinstitute" %% "cromwell-wdl" % cromwellVersion,
+  "org.broadinstitute.dsde.vault" %% "vault-common" % "0.1-15-bf74315" excludeAll ExclusionRule(organization = "io.spray"),
```

Comment:
What's the vault dependency for?

Reply:
+1 to this question. Vault was specifically removed in prep for moving to scala 2.12. See https://github.com/broadinstitute/agora/commit/43b1dfb114f74b6fab5b67962666990a70e5f8c6